HOW_TO_TEST.md

```diff
@@ -1,97 +1,115 @@
 # How to test
 
-Starting with CP-Ansible 5.5.0, we have included testing via the [Molecule](https://molecule.readthedocs.io/en/latest/) framework, and strongly advise its usage before submitting a Pull Request.
+CP-Ansible's tests use the [Molecule](https://molecule.readthedocs.io/en/latest/) framework, and it is strongly advised to test this way before submitting a Pull Request.
 
 ## Prerequisites
 
-1. Python3 installed with PIP
-2. Docker installed
-
-Note: We recommend increasing your docker memory to at least 20GB of RAM and your CPU count to 10.  
-
-### Installing Molecule via pip
-
-To install the Molecule and Docker libraries  
-
-```pip install molecule docker```
-
-Note: If using Molecule version 3.1.0 or later use the following command to install:
-
-```pip3 install molecule molecule-docker```
-
-
-### Running Molecule in a Container
-
-At times, pip installation of molecule can lead to errors:
+1. Python3
+2. [Ansible >= 2.11 installed with Pip](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-ansible-with-pip). Use python3's pip for ansible installation to make sure ansible is not configured with Python2.7:
 ```
-ImportError: No module named docker.common
+python3 -m pip install --user ansible
 ```
-
-As a workaround you can use molecule in a container.  
-In your current shell create an alias to start molecule in a container:
-
+2. [Docker](https://docs.docker.com/get-docker/) *Note: We recommend increasing your docker memory to at least 20GB of RAM and your CPU count to 10.*
+3. [Molecule](https://molecule.readthedocs.io/en/latest/installation.html#install). Use python3's pip for installation ie 
```

Comment:
I think we have a requirement on the Molecule version. At least 3.x.

Reply:
nice catch

Reply:
```suggestion
3. [Molecule >= 3.3](https://molecule.readthedocs.io/en/latest/installation.html#install). Use python3's pip for installation: 
```